### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by YevhenDov**

### CODE REVIEW

File: YouTubeParser/src/main/java/com/model/Comment.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
+package com.model;
+
+import com.google.api.client.util.DateTime;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+
```
Comment (Anderwerty) [inline YouTubeParser/src/main/java/com/model/Comment.java]: Redundant blank line.

File: YouTubeParser/src/main/java/com/model/Comment.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+public class Comment {
```
Comment (Anderwerty) [inline YouTubeParser/src/main/java/com/model/Comment.java]: Why not create a builder for this class?

File: YouTubeParser/src/main/java/com/parser/Auth.java [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+        FileDataStoreFactory fileDataStoreFactory =
+                new FileDataStoreFactory(new File(System.getProperty("user.home") + "/" + CREDENTIALS_DIRECTORY));
+        DataStore<StoredCredential> dataStore = fileDataStoreFactory.getDataStore(credentialDataStore);
+
+        GoogleAuthorizationCodeFlow flow = new GoogleAuthorizationCodeFlow.Builder(
+                HTTP_TRANSPORT, JSON_FACTORY, clientSecrets, scopes).setCredentialDataStore(dataStore)
+                .build();
+
+        LocalServerReceiver localReceiver = new LocalServerReceiver.Builder().setPort(8080).build();
```
Comment (Anderwerty) [inline YouTubeParser/src/main/java/com/parser/Auth.java]: 8080 looks like a magic number. Why not use a constant?

File: YouTubeParser/src/main/java/com/parser/CommentParser.java [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+    private YouTube youTube;
+    private static List<String> scopes = Lists.newArrayList("https://www.googleapis.com/auth/youtube.force-ssl");
+    private YouTubeVideo youTubeVideo = new YouTubeVideo();
+
+    public void init() {
+        Credential credential = null;
+        try {
+            credential = Auth.authorize(scopes, "CommentThreads");
+        } catch (Exception e) {
+            e.printStackTrace();
```
Comment (Anderwerty) [inline YouTubeParser/src/main/java/com/parser/CommentParser.java]: Why you don't use logger here?

File: YouTubeParser/src/main/java/com/parser/CommentParser.java [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+                    commentList.add(comment);
+                }
+                youTubeVideo.setCommentList(commentList);
+                youTubeVideo.setVideoId(videoId);
+
+                System.out.println(youTubeVideo.getVideoId());
+                System.out.println(youTubeVideo.getCommentList());
+
+            } catch (Exception e) {
+                e.printStackTrace();
```
Comment (Anderwerty) [inline YouTubeParser/src/main/java/com/parser/CommentParser.java]: The same

File: YouTubeParser/src/main/java/com/parser/CommentParser.java [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+            }
+        }
+    }
+}
+
```
Comment (Anderwerty) [inline YouTubeParser/src/main/java/com/parser/CommentParser.java]: A redundant blank line at the end of the file.

File: YouTubeParser/src/main/resources/client_secrets.json [hunk 7]
```diff
@@ -0,0 +1,14 @@
+{
```
Comment (Anderwerty) [inline YouTubeParser/src/main/resources/client_secrets.json]: It is not a good idea to publish credentials into the public repository. 
Better add this file to gitignore and remove from repo.

File: YouTubeParser/src/main/java/com/model/Comment.java (hunk 2 above)
Comment (YevhenDov) [reply YouTubeParser/src/main/java/com/model/Comment.java]: Setters is more obvious

**Pull request merged**